pyaedt/maxwell.py

```diff
@@ -771,8 +800,8 @@ def assign_rotate_motion(
         damping : float, optional
             Damping factor. The default is ``0``.
         load_torque : float or str, optional
-            Load force. The default is ``"0newton"``. If a float value is used,
-            "NewtonMeter" units are applied.
+            Load torque sign is determined based on the moving vector, using the right-hand rule.
+            The default is ``"0newton"``. If a float value is used "NewtonMeter" units are applied.
```

Comment:
If it is a torque, shouldn't the default value be expressed in "NewtonMeter" too?
```suggestion
            The default is ``"0NewtonMeter"``. If a float value is used "NewtonMeter" units are applied.
```

Reply:
Is the word "sign" in the description necessary? I just don't know your terminology--but it seems odd.